lib/src/generator/template_data.dart

```diff
@@ -384,7 +385,8 @@ class MethodTemplateData extends TemplateData<Method>
 
   MethodTemplateData(super.htmlOptions, super.packageGraph, this.library,
       this.container, this.method, this._sidebarForContainer)
-      : _containerDescription = container.isClass ? 'class' : 'extension';
+      : _containerDescription =
+            container is ClassElement ? 'class' : 'extension';
```

Comment:
What if we get `MixinElement` here?
Should we check for `InterfaceElement` instead?

Reply:
lol technical debt. Fixed to InterfaceElement and added a TODO.